frame/recovery/src/mock.rs

```diff
@@ -80,7 +80,7 @@ impl frame_system::Trait for Test {
 	type ModuleToIndex = ();
 	type AccountData = pallet_balances::AccountData<u128>;
 	type OnNewAccount = ();
-	type OnReapAccount = (Balances, Recovery);
+	type OnReapAccount = Balances;
```

Comment:
Just a mock, but still raises the question: The balances is declared in this dummy runtime to use the system as storage. Does it need to be on then `OnReapAccount ` then?

Reply:
You are probably right here. let me look

Reply:
This was misleading then `type AccountStore = System;`

Reply:
Ah I think I might be wrong then...

Reply:
Yes, I removed them. thanks!